crates/catalog/sql/src/error.rs

```diff
@@ -32,3 +32,20 @@ pub fn no_such_namespace_err<T>(namespace: &NamespaceIdent) -> Result<T> {
         format!("No such namespace: {:?}", namespace),
     ))
 }
+
+pub fn no_such_table_err<T>(table_ident: &TableIdent) -> Result<T> {
+    Err(Error::new(
+        ErrorKind::Unexpected,
+        format!("No such table: {:?}", table_ident),
+    ))
+}
+
+pub fn table_already_exists_err<T>(table_ident: &TableIdent) -> Result<T> {
+    Err(Error::new(
+        ErrorKind::Unexpected,
+        format!(
+            "Cannot create table {:?}. Table already exists.",
```

Comment:
nit: this error is used in other functions such as rename. 
```suggestion
            "Table already exists: {:?}",
```

Reply:
This is a good point, thank you, fixed.